cypress/support/fragments/inventory/inventoryViewSource.js

```diff
@@ -77,4 +78,22 @@ export default {
         });
     }
   },
+
+  verifyLinkedToAuthorityIcon(rowIndex) {
+    cy.expect(
+      rootSection
+        .find(TableRow({ index: rowIndex }))
+        .find(linkedToMarcAuthorityIcon)
+        .exists(),
+    );
+  },
+
+  verifyLinkedToAuthorityIconAbsence(rowIndex) {
+    cy.expect(
+      rootSection
+        .find(TableRow({ index: rowIndex }))
+        .find(linkedToMarcAuthorityIcon)
+        .absent(),
+    );
+  },
```

Comment:
what if we combine these two methods and check the absence and existence according to a passed argument like: 
verifyLinkedToAuthorityIcon(rowIndex, exists = true)